### ISSUE TITLE
timely board agendas and records; else appeal / cancel

### ISSUE THREAD

**Issue by dckc** [rchain-community/sm19a#11]
polling: [9:49 PM KC time in #governance May 5](https://discordapp.com/channels/375365542359465989/393462637100400650/574789795063267329)

_Note: this proposal would be stronger with an offer from a member to assist in preparation of board meeting agendas and records._

Members of the board are elected by the membership and hence represent their interests. The board should
  - Invite input from the membership on items of business
  - be available to collect advice from the membership on these items
  - announce decisions in a timely manner

I propose that **board meeting minutes must be available to the membership within 2 weeks** or the meeting is retroactively cancelled, nullifying any decisions taken.

In some circumstances, the RChain cooperative is better served by having board members keep business confidential until a decision is made. Occasionally, proposals may be generated or amended during a meeting. But in these cases, it is especially important that the decision have the support of the membership.

**Any resolution not part of an agenda shared with the membership 48 hours in advance must be shared with the membership within 48 hours after the start of the meeting and is subject to appeal** by the governance committee, audit and finance committee, executive committee, and nominating committee; that is: should any of these committees decide (within three weeks of the start of the board meeting time) to appeal such a decision, it shall have up to one week following its decision to appeal to board members to reconsider their positions. If the decision does not have a 2/3rds majority of the board membership at the end of this one week period, the decision is vacated.

 For reference: [issues/564](https://github.com/rchain/bounties/issues/564) shows my struggles to get timely access to board minutes since Apr 2018.

See also [Barry 25 Oct 2018 6:50 PM KC time](https://discordapp.com/channels/375365542359465989/396078336620888065/505166347114643475)

**Comment by nmeyne** [comment]
This should be written into the Charter of the Main Board.   Suggest that GP001 is also approved alongside this so that:
1) appropriate guidance is given on what a coherent 'significant' proposal should look like (e.g. rationale,  alternatives considered, risk, impact and so on) as opposed to a simple or trivial resolution.
2) the appeals process envisaged in GP001 can be invoked by the committees (or any member, if they have support)
3) encourages the discipline of the main board approving stuff that has been thought through, discussed and is shown to be supported BEFORE being brought to the meeting.  NOT discussed and cooked up 'on the fly' or brought in as a 'surprise' unless in genuine extremis.

**Comment by JoshOrndorff** [comment]
> I propose that board minutes must be available to the membership within 2 weeks of each meeting or the meeting is retroactively cancelled, nullifying any decisions taken.

I like this idea because it has an `or else` clause. As opposed to just demanding that the minutes be published or else nothing. We've seen how that works.

**Comment by dckc** [comment]
@nmeyne I agree [GP001][] is relevant, but I think it's not necessary to specify what is trivial vs. what is significant for this issue. The question of whether something is significant is delegated to the committees. GP001 is a useful tool for the membership to understand how the governance committee, at least, intends to make the determination.

[GP001]: https://docs.google.com/document/d/1nBRLjw8tuQd05GIacIhrN2FwFPJInwaiLNwiW0nb-E4/edit#
